src/pudl/metadata/sources.py

```diff
@@ -21,6 +21,45 @@
         "license_raw": LICENSES["us-govt"],
         "license_pudl": LICENSES["cc-by-4.0"],
     },
+    "eia176": {
+        "title": "EIA Form 176",
+        "path": "https://www.eia.gov/naturalgas/ngqs/all_ng_data.zip",
```

Comment:
IIRC and looking at the other sources it seems like the `path` is meant as a human reference, rather than the actual data URL, so I think we probably want to use https://www.eia.gov/naturalgas/ngqs/ here.

Reply:
Great point. Done & done.